src/components/customizableAside.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import CustomizableAvatar from "./common/customavatar/CustomizableAvatar"
+
+function CustomizableAside({asideCustomClass, src, height, width, alt, paragraphText, headerText}) {
```

Comment:
Now I have merged your `Aside` component, there is no need for this. Just import it.